### PR TITLE
initialized the website with rails and mongoDB

### PR DESCRIPTION
**Pull request by xiuyu-lana**

### CODE REVIEW

File: config/initializers/content_security_policy.rb
Comment (nilo-tamu) [inline config/initializers/content_security_policy.rb:1]: What is the purpose of this file?

**Review by nilo-tamu — CHANGES_REQUESTED** [review]
As per team conversation, holding off on PR approval until unit tests are passing (please attach test evidence in PR body).

**Comment by nilo-tamu** [comment]
In the future we can also revise the necessity of some of these files in the code base so that it becomes cleaner.

**Comment by xiuyu-lana** [comment]
Two modifications: 1. add validations for the game title and URL; 2. add tests for models, controllers, views, and requests (under LIVE/spec folder). To run all tests, run `rspec` under LIVE folder. As shown in the image below, all tests passed. Please review@nilo-tamu 
IMG_6507

**Comment by nilo-tamu** [comment]
Also are there Cucumber tests in here? Trying to observe if that is something that could be inserted at this point or if it should come ion later

**Review by nilo-tamu — APPROVED** [review]

**Pull request merged**